lib/features/composer/presentation/composer_controller.dart

```diff
@@ -721,30 +727,58 @@ class ComposerController extends BaseController {
     }
   }
 
-  Future<List<EmailAddress>> getAutoCompleteSuggestion(
-      {required String word}) async {
-    log('ComposerController::getAutoCompleteSuggestion(): $word | $_contactSuggestionSource');
+  @override
+  void injectAutoCompleteBindings(Session? session, AccountId? accountId) {
+    if (session == null || accountId == null) {
+      return;
+    }
+    try {
+      super.injectAutoCompleteBindings(session, accountId);
+      _getAutoCompleteWithDeviceContactInteractor = Get.find<GetAutoCompleteWithDeviceContactInteractor>();
+      _getAutoCompleteInteractor = Get.find<GetAutoCompleteInteractor>();
```

Comment:
How about the app run on the web? (no GetAutoCompleteWithDeviceContactInteractor but GetAutoCompleteInteractor), exception will throw after line 737?

Reply:
Don't worry, `_getAutoCompleteWithDeviceContactInteractor` always injected.